osu.Game/Screens/SelectV2/FilterControl.cs

```diff
@@ -274,6 +277,15 @@ private partial class SongSelectSearchTextBox : ShearedFilterTextBox
             private partial class InnerTextBox : InnerFilterTextBox
             {
                 public override bool HandleLeftRightArrows => false;
+
+                public override bool OnPressed(KeyBindingPressEvent<PlatformAction> e)
+                {
+                    // the "cut" platform key binding (shift-delete) conflicts with the beatmap deletion action.
+                    if (e.Action == PlatformAction.Cut && e.ShiftPressed && e.CurrentState.Keyboard.Keys.IsPressed(Key.Delete))
+                        return false;
+
+                    return base.OnPressed(e);
+                }
```

Comment:
Copied from old song select, see `FilterControl.FilterControlTextBox`. Still suffering from the same issue as that.